### PR TITLE
chore: setup dev environment

### PR DESCRIPTION
**Pull request by antoniomuso**
## 🚨 Proposed changes

I am doing the setup of dev environment

## ⚙️ Types of changes

What types of changes does your code introduce? _Put an `x` in the boxes that apply_

-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Documentation Update (if none of the other choices apply)
-   [ ] Refactor

### CODE REVIEW

File: app-config.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    # Default Helmet Content-Security-Policy values can be removed by setting the key to false
+  reading:
+    allow:
+      - host: example.com
+      - host: '*.mozilla.org'
+  # workingDirectory: /tmp # Use this to configure a working directory for the scaffolder, defaults to the OS temp-dir
+
+integrations:
+  gitlab:
+    - host: gitlab.rete.farm
```
Comment (JellyBellyDev) [inline app-config.yaml:54]: change please!!!!

File: tsconfig.json [hunk 2]
```diff
@@ -1,8 +1,13 @@
...
+    "exclude": ["node_modules"],
     "compilerOptions": {
-        "outDir": "dist-types/src",
-        "incremental": false
+      "outDir": "dist-types",
+      "rootDir": "."
     }
-}
+  }
+
```
Comment (JellyBellyDev) [inline tsconfig.json:13]: new line at end of file! ;)

**Comment by antoniomuso** [comment]
To test UI in an easy way. You can spawn an instance of UI components using `yarn start`

**Comment by antoniomuso** [comment]
This is an example of `yarn start`. I am still ending to put the mocked data.
image

**Comment by JellyBellyDev** [comment]
Nice catch! 🚀

**Comment by JellyBellyDev** [comment]
I suggest you to write a contributing.md file with all steps to work on it!

**Review by JellyBellyDev — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -47,10 +47,15 @@
     "peerDependencies": {
         "react": "^16.13.1 || ^17.0.0",
         "react-dom": "^16.13.1 || ^17.0.0",
-        "react-router-dom": "6.0.0-beta.0 || ^6.3.0",
-        "react-router": "6.0.0-beta.0 || ^6.3.0"
+        "react-router": "6.0.0-beta.0 || ^6.3.0",
```
[the remark this answers is not in the document]
Comment (antoniomuso) [reply package.json:50]: Until backstage does not deprecate the beta, we have to support it.

File: src/api/GitlabCIClient.ts [hunk 4]
```diff
@@ -18,24 +18,28 @@ import {
 export class GitlabCIClient implements GitlabCIApi {
     discoveryApi: DiscoveryApi;
     baseUrl: string;
+    proxyPath: string;
     constructor({
         discoveryApi,
         baseUrl = 'https://gitlab.com/',
+        proxyPath = '/gitlabci',
```
[the remark this answers is not in the document]
Comment (antoniomuso) [reply src/api/GitlabCIClient.ts:25]: It is the path for the proxy; until now was hardcoded.

**Pull request merged**